src/main/resources/templates/create.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Since we can include files, maybe we should create the following pages : 
- header.html (contains `<html><head></head><body>` and your "base" view) so we don't need to duplicate the html tag on each page
- footer.html (contains `</body></html>`)
And then all pages just need to include the header and the body